benefits/core/views.py

```diff
@@ -56,19 +56,15 @@ def agency_index(request, agency):
         return redirect(reverse(ROUTE_ELIGIBILITY))
 
     button = viewmodels.Button.primary(text=_("core.pages.index.continue"), url=reverse(ROUTE_ELIGIBILITY))
-    button.label = _("core.pages.agency_index.button.label")
 
     page = viewmodels.Page(
         title=_("core.pages.agency_index.title"),
-        content_title=_("core.pages.agency_index.content_title"),
+        content_title=_("core.pages.agency_index.mst_cc.content_title"),
```

Comment:
Just so I'm following - this is where we'll need to dynamically pick the title in the future, right?

Reply:
Yeah